ceno_zkvm/src/chip_handler/general.rs

```diff
@@ -73,15 +73,16 @@ impl<'a, E: ExtensionField> CircuitBuilder<'a, E> {
         &mut self,
         name_fn: N,
         table_len: usize,
-        rlc_record: Expression<E>,
+        rom_type: ROMType,
+        items: Vec<Expression<E>>,
```

Comment:
What's in `items`?  Could you add a dock comment or so, please?

Reply:
value of the original record before doing RLC. This is renamed to `record`.